examples/fineFoods/admin/mui/src/components/store/storeProducts.tsx

```diff
@@ -0,0 +1,300 @@
+import React from "react";
+import { useTranslate, useTable, useUpdate } from "@pankod/refine-core";
+import { useModalForm } from "@pankod/refine-react-hook-form";
+import {
+    Grid,
+    Paper,
+    Typography,
+    Box,
+    InputBase,
+    IconButton,
+    Stack,
+    Pagination,
+    Button,
+    Modal,
+    Fade,
+} from "@pankod/refine-mui";
+import { SearchOutlined, CloseOutlined } from "@mui/icons-material";
+
+import {
+    CategoryFilter,
+    ProductItem,
+    CreateProduct,
+    EditProduct,
+} from "components";
+import { IStore, IProduct } from "interfaces";
+
+type StoreProductsProps = {
+    record: IStore;
+    close: () => void;
+    visible: boolean;
+};
+
+export const StoreProducts: React.FC<StoreProductsProps> = ({
+    record,
+    close: modalClose,
+    visible: modalVisible,
+}) => {
+    const t = useTranslate();
+
+    const {
+        modal: { visible, close, show, saveButtonProps },
+        register,
+        refineCore: { onFinish },
+        formState: { errors },
+        handleSubmit,
+        watch,
+        setValue,
+        control,
+    } = useModalForm<IProduct>({
+        refineCoreProps: { action: "create" },
+    });
```

Comment:
```suggestion
        modal: { visible, close, show, saveButtonProps },
        register,
        refineCore: { onFinish },
        formState: { errors },
        handleSubmit,
        watch,
        setValue,
        control,
    } = useModalForm<IProduct>({
        refineCoreProps: { action: "create" },
    });
```
maybe we can use `create` prefix for props.
like, `createDrawerVisible`